### DISCUSSION TITLE
Teams

### DISCUSSION THREAD

**Discussion by GustavoLoreto** [Launch-X-Latam/MisionFrontEnd#535] [Team Posts]
Oigan, porque mejor no se ponen de acuerdo para los equipos en Teams? Mi correo ya está full.
Por otro lado, desde el incio nos comentaron que no estaban permitidos los canales de Discord, Grupos de Whats etc.

**Comment by agenciakolob** [comment]
Para que no te llegue muchos correos puedes cambiar los ajustes en GitHub>Ajustes>Notificaciones
